macros/create_base_models.sql

```diff
@@ -0,0 +1,16 @@
+{% macro create_base_models(source_name, tables) %}
+
+{% set source_name = ""~ source_name ~"" %}
+
+{% set zsh_command_models = "source dbt_packages/codegen/bash_scripts/base_model_creation.sh """~ source_name ~""" " %}
+
+{%- set models_array = [] -%}
+
+{% for t in tables %}
+    {% set help_command = zsh_command_models + t %}
+    {{ models_array.append(help_command) }}
+{% endfor %}
+
+{{ log(columns_array|join(' && \n') + ' && \n' + models_array|join(' && \n'), info=True) }}
```

Comment:
Is `columns_array` vestigial? I believe it is causing the extra `&&` noted [here](https://github.com/dbt-labs/dbt-codegen/pull/83#issuecomment-1344703364).

Suggesting the following to remove `columns_array` and `&&` and add some light commentary.

I'm intending to just proactively commit this:
```suggestion
{{ log("Run these commands in your shell to generate the models:\n" ~ models_array|join(' && \n'), info=True) }}
```

This will end up with output like this that keeps it easy to see all three commands to copy-paste:
```shell
19:41:24  Running with dbt=1.3.1
19:41:24  Run these commands in your shell to generate the models:
source dbt_packages/codegen/bash_scripts/base_model_creation.sh dbt_dbeatty raw_orders && 
source dbt_packages/codegen/bash_scripts/base_model_creation.sh dbt_dbeatty raw_customers && 
source dbt_packages/codegen/bash_scripts/base_model_creation.sh dbt_dbeatty asdfasdf
```